src/Module/Settings/Channels.php

```diff
@@ -232,4 +238,21 @@ protected function content(array $request = []): string
 			'$form_security_token' => self::getFormSecurityToken('settings_channels'),
 		]);
 	}
+
+	private function enableTimeline(int $uid, int $id)
+	{
+		$bookmarked_timelines = $this->pConfig->get($uid, 'system', 'network_timelines');
+		$enabled_timelines    = $this->pConfig->get($uid, 'system', 'enabled_timelines');
+
+		if (empty($enabled_timelines) || empty($bookmarked_timelines)) {
```

Comment:
Doesn't this mean channels will not be enabled if none were enabled before?

Reply:
When it hadn't been stored in that field before, the system will always display all entries by default.